website/src/client/configs/defaults.tsx

```diff
@@ -96,11 +96,11 @@ const styles = StyleSheet.create({
 
 Open the \`App.js\` file to start writing some code. You can preview the changes directly on your phone or tablet by scanning the **QR code** or use the iOS or Android emulators. When you're done, click **Save** and share the link!
 
-When you're ready to see everything that Expo provides (or if you want to use your own editor) you can **Download** your project and use it with [expo-cli](https://docs.expo.io/get-started/installation).
+When you're ready to see everything that Expo provides (or if you want to use your own editor) you can **Download** your project and use it with [Expo CLI](/get-started/installation/#expo-cli).
```

Comment:
Since we are on `snack.expo.dev`, I think you still need to add the domain.

```suggestion
When you're ready to see everything that Expo provides (or if you want to use your own editor) you can **Download** your project and use it with [Expo CLI](https://docs.expo.dev/get-started/installation/#expo-cli).
```

Reply:
My bad. That was out of habit 😅